build.sbt

```diff
@@ -322,7 +322,7 @@ val docs = project
     name := "bloop-docs",
     moduleName := "bloop-docs",
     skip in publish := true,
-    scalaVersion := "2.12.6",
+    scalaVersion := "2.12.9",
```

Comment:
Maybe replace by the almost always up-to-date `Dependencies.Scala212Version`? We can update the codebase to 2.12.9 in another PR.